frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizationsContainer.test.tsx

```diff
@@ -488,7 +488,12 @@ describe("PendingOrganizationsContainer", () => {
           await waitForElementToBeRemoved(
             screen.queryByText("Organization details")
           );
-          expect(screen.getByText("DC Space Camp")).toBeInTheDocument();
+
+          expect(
+            screen.getByRole("heading", {
+              name: /dc space camp details updated/i,
```

Comment:
Is this change related to this PR? If not could you make a separate change for it?

Reply:
I added this change because the frontend unit test randomly started failing with an error of not being able to find the text "DC Space Camp" so in order to pass the unit test I had to change it.

Reply:
This test was only failing in the CI (local always passed the test). Seems that root cause was that jest was not properly waiting for the DOM to be updated and failing the test too soon. I have restored the former assertion but added waiting code to do the assertion async.